src/Raven.Server/Documents/PeriodicBackup/Azure/RavenAzureClient.cs

```diff
@@ -590,6 +590,10 @@ public void DeleteMultipleBlobs(List<string> blobs)
             if (blobs.Count == 0)
                 return;
 
+            DeleteBlobsWithSasToken(blobs);
+
+            //TODO: RavenDB-16264
+            /*
```

Comment:
Why do we keep so much dead code?

Reply:
This is not entirely dead code. We want to support multi-delete, but it requires further investigation so for now it is better to go back to a one by one call as I understood from @garayx and @grisha-kotler